File: accomondationApp/Models/Reservation.cs
using System;
using System.Collections.Generic;

namespace accomondationApp.Models;

public partial class Reservation
{
    public int ReservationId { get; set; }

    public int? RommId { get; set; }

    public DateTime? StartDate { get; set; }

    public DateTime? EndDate { get; set; }

    public int? CustomerId { get; set; }

    public int? PaymentStatusId { get; set; }

    public virtual Customer? Customer { get; set; }

    public virtual PaymentStatus? PaymentStatus { get; set; }

    public virtual Room? Romm { get; set; }
}

File: accomondationApp/ClientApp/src/app/interfaces/Reservation.ts
import { DateTime, Info, Interval} from 'luxon';
import { Room } from "../interfaces/Room"
import { Customer } from './Customer';
import { PaymentStatus } from './PaymentStatus';

export interface Reservation{
  reservationId: number,
  rommId: number,
  customer: Customer,
  romm: Room,
  paymentStatusId: number,
  startDate: DateTime,
  endDate: DateTime,
  name: string,
  email: string,
  days: any [],
  reservationDate: DateTime
}
